web/pdf_thumbnail_view.js

```diff
@@ -405,11 +423,13 @@ var PDFThumbnailView = (function PDFThumbnailViewClosure() {
     },
   };
 
+  PDFThumbnailView.cleanup = function() {
```

Comment:
Since the temporary canvas is only used by `PDFThumbnailView`, the `PDFThumbnailViewer` really shouldn't know anything about them (the current code isn't correct in my opinion).

What you're commenting on is a static method, i.e. one that is shared between all `PDFThumbnailView`s, so there's nothing to iterate over here!